src/pages/LandingPages/MGCovid19/MGCovidHero.vue

```diff
@@ -152,6 +152,8 @@ export default {
 				['wga retina', heroImagesRequire('./hero_3840x1480.jpg')],
 			],
 			expVideoActive: true,
+			isMonthlyGoodSubscriber: false,
```

Comment:
This isn't related to the LBC homepage but I saw this error popping up.

Reply:
should be fine as it just defines these in data.